pandas/core/internals/blocks.py

```diff
@@ -1056,8 +1046,7 @@ def coerce_to_target_dtype(self, other):
                 return self.astype(object)
 
             raise AssertionError(
-                "possible recursion in "
-                "coerce_to_target_dtype: {} {}".format(self, other)
+                "possible recursion in " f"coerce_to_target_dtype: {self} {other}"
```

Comment:
an extra space snuck in the middle here (this is a common occurrence with black)